app/grants/templates/grants/cancel.html

```diff
@@ -20,7 +20,6 @@
   <head>
     {% include 'shared/head.html' %}
     {% include 'shared/cards.html' %}
-    <link rel="stylesheet" href="{% static "v2/css/bounty.css" %}">
```

Comment:
@captnseagraves this needs to added back in as the waiting loader needs css from this section.